scala/scala-impl/src/org/jetbrains/plugins/scala/testingSupport/test/ScalaTestRunLineMarkerProvider.scala

```diff
@@ -64,7 +64,7 @@ class ScalaTestRunLineMarkerProvider extends TestRunLineMarkerProvider {
     case _ => false
   }
 
```

Comment:
I decided to reuse this method, If we decide that using `ScalaTestRunLineMarkerProvider` is relevant and we can extend it, then getInfo needs to be protected for usability.